pkg/chain/ethereum/tbtc.go

```diff
@@ -158,6 +158,46 @@ func (tec *TBTCEthereumChain) OnDepositRedeemed(
 	).OnEvent(onEvent)
 }
 
+// PastCreatedEvents returns all created events for the given deposit which
+// occurred after the provided start block. Returned events are sorted by the
+// block number in the ascending order.
+func (tec *TBTCEthereumChain) PastCreatedEvents(
```

Comment:
I think we should include `Deposit` in the name. This will also be consistent with another function we've got in this file `PastDepositRedemptionRequestedEvents`.
```suggestion
func (tec *TBTCEthereumChain) PastDepositCreatedEvents(
```